packages/amplify-category-auth/resources/cloudformation-templates/auth-template.yml.ejs

```diff
@@ -182,12 +185,23 @@ Resources:
       AliasAttributes: !Ref aliasAttributes
       <% } %>
       MfaConfiguration: !Ref mfaConfiguration
+      <% if(props.useEnabledMfas && props.mfaConfiguration != 'OFF') {%>
+      EnabledMfas:
+        <% if(configureSMS) {%> 
+        - SMS_MFA
+        <% } %>
+        <% if(props.mfaTypes.includes('TOTP')) {%>
+        - SOFTWARE_TOKEN_MFA
+        <% } %>
+      <% } %>  
+      <% if(!props.useEnabledMfas || configureSMS) {%> 
```

Comment:
Are we not using these messages when mfa is false and configureSMS is true?

Reply:
The feature flag is to retain the current state of existing projects. which always keeps SMS configured

Reply:
For new project we only configure the SMS when they start using phone_numbers as a required attribute or an auto verified attribute